build.gradle.kts

```diff
@@ -12,6 +12,19 @@ java {
     toolchain.languageVersion = JavaLanguageVersion.of(8)
 }
 
+dependencies.constraints {
+    checkstyle("com.google.guava:guava") {
+        version {
+            require("33.3.1-jre")
+        }
+    }
+    testSamplesImplementation("commons-io:commons-io") {
+        version {
+            require("2.18.0")
+        }
+    }
+}
```

Comment:
For better readability, I would use the standard short notations, like `com.google.guava:guava:33.3.1-jre`

Reply:
If I do that, it's automatically a require?

Reply:
Yes. It is the short notation for `require`.